health/target/target.go

```diff
@@ -0,0 +1,55 @@
+// Package target provides exported structs that you should use during
```

Comment:
Confusing to have two files named target in the health module. Please rename one....

Reply:
Fixed: /health/target.go -> /health/registry.go